### PR TITLE
Add TLS configuration for API endpoint

### PR DESCRIPTION
**Pull request by davewalter**
## Is there a related GitHub Issue?
#161 

## What is this change about?
This PR adds configuration for a TLS certificate for the API endpoint and updates the README to tell the user how to create the required certificate and secret.

## Does this PR introduce a breaking change?
No

## Acceptance Steps
See #161

## Tag your pair, your PM, and/or team
@acosta11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,14 +44,48 @@ You can deploy the app to your cluster by running `make deploy` from the project
...
+  -nodes -subj '/CN=localhost' \
+  -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf "[ SAN ]\nsubjectAltName='DNS:localhost'")) \
+  -days 365
+```
+
+Create a TLS secret called `cf-k8s-api-ingres-cert` using the self-signed
+certificate generated above, or from your own existing certificate:
+```
+kubectl create secret tls \
+  cf-k8s-api-ingres-cert \
```
[the remark this answers is not in the document]
Comment (davewalter) [reply README.md]: Fixed.

**Comment by davewalter** [comment]
> Does adding this config disable `http` for the API endpoint or does Contour serve both still?

I think `http` will redirect to `https`.

**Review by tcdowney — APPROVED** [review]
lgtm

**Comment by davewalter** [comment]
Closing as it has been recreated as https://github.com/cloudfoundry/cf-k8s-controllers/pull/137

**Pull request closed without merging**